pkg/tsdb/Magefile.go

```diff
@@ -26,8 +26,16 @@ func find(dir string, name string) ([]string, error) {
 	return files, err
 }
 
-func findPluginJSONDir(pluginDir string) (string, error) {
-	pluginJSONMatches, err := find(filepath.Join("../../public/app/plugins/datasource", pluginDir), "plugin.json")
+func fileHasString(path string, s string) bool {
+	f, err := os.ReadFile(path)
+	if err != nil {
+		return false
+	}
+	return strings.Contains(string(f), s)
+}
+
+func findPluginJSONDir(pluginID string) (string, error) {
+	pluginJSONMatches, err := find(filepath.Join("../../public/app/plugins/datasource"), "plugin.json")
```

Comment:
Nit: Maybe something to use eventually here instead https://pkg.go.dev/path/filepath#Glob

Reply:
right, a pity that Glob doesn't work with `**` in paths